src/app/forms/petition-for-conviction/petition-for-conviction.pdf.component.js

```diff
@@ -121,14 +121,14 @@ export default class PetitionForConviction_Pdf extends React.Component {
             debugKey="caseCourtType"
             left="37.5%"
             top="36.5%"
-            dataKey="case.districtCourt"
```

Comment:
We should do `dataKey` when we are showing data and `debugKey` when we have a container that calls `renderData()` inside of it. In this case, we should use `dataKey`.